requirements.txt

```diff
@@ -26,3 +26,6 @@ influxdb==5.3.0
 jsonschema==3.2.0
 orjson==3.4.5
 h5py==2.10.0
+pyspellchecker==0.6.1
```

Comment:
I would remove "pyspellchecker" if it's never used.
+ the package "joblib" seems to be required but is not in the list.